<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PHX-ISHARAF" ignored="2" name="Branch Opportunity Test Scenarios" tests="7" failures="0" timestamp="16 Dec 2019 17:09:07 GMT" time="441.617" errors="0">
  <testcase name="createBranchOptty_viaAPI" time="31.578" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="moveStageStatusManually" time="41.528" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="opttyClosedLost_viaAPI" time="51.11" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="logacall_notReached_Optty" time="57.663" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="logacall_Reached_StageClosedLost" time="65.264" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="scheduleMeetingandthenCloseMeeting" time="63.023" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="scheduleMeetingviaNextAction" time="86.639" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="logacall_Reached_Optty" time="0.0" classname="com.crm.qa.testcases.BranchOpportunityTest">
    <ignored/>
  </testcase> <!-- logacall_Reached_Optty -->
  <testcase name="exp" time="0.0" classname="com.crm.qa.testcases.BranchOpportunityTest">
    <ignored/>
  </testcase> <!-- exp -->
</testsuite> <!-- Branch Opportunity Test Scenarios -->
